backend/requirements.txt

```diff
@@ -6,4 +6,8 @@ Requests==2.31.0
 uvicorn==0.22.0
 python-multipart
 sqlalchemy
-psycopg2
\ No newline at end of file
+psycopg2
+PyPDF2
+pytesseract
+Pillow
+cv2
```

Comment:
I had an error doing pip install, regarding cv2 (ERROR: Could not find a version that satisfies the requirement cv2 (from versions:none) ERROR: No matching distribution found for cv2.)
Some googling suggests that some different versions may be required, e.g. opencv-python?

Reply:
opencv-python package needs to be install. fixed in new update